glommio/src/read_write_lock.rs

```diff
@@ -43,14 +43,16 @@ use std::future::Future;
 use std::ops::{Deref, DerefMut};
 use std::pin::Pin;
 use std::task::{Context, Poll, Waker};
+use std::io;
+use futures_lite::io::ErrorKind;
```

Comment:
any particular reason we're using `ErrorKind` from `futures_lite` instead of `std::io::ErrorKind` ?

Reply:
Hi @glommer it is already fixed in the latest commit. I suppose you missed it. IntelliJ automatically inserted the use of futures_lite, I have changed that already.